internal/pkg/cli/job_deploy.go

```diff
@@ -46,7 +46,7 @@ type deployJobOpts struct {
 	targetApp       *config.Application
 	targetEnv       *config.Environment
 	envSess         *session.Session
-	appliedManifest interface{}
+	appliedManifest manifest.WorkloadManifest
```

Comment:
This reminds me that we probably want to rename `manifest.WorkloadManifest` to `manifest.DynamicWorkloadManifest`: the interface now requires the implementation of `Load()`, which makes it dynamic. The reason why I thought of this was because I found myself having a hard time trying to wrap my head around on how `DynamicWorkloadManifest` implements `WorkloadManifest`, while it has a member field that's of type `workloadManifest`  which actually doesn't implement `WorkloadManifest` though spelled the same  😵